src/components/navigation-header/index.tsx

```diff
@@ -35,12 +34,13 @@ const RightSideHeaderContent = () => {
  */
 const NavigationHeader = () => {
   const { isDesktop } = useDeviceSize()
-  const currentPath = useCurrentPath({ excludeHash: true, excludeSearch: true })
-  const LeftSideHeaderContent =
-    currentPath === '/' ? HomePageHeaderContent : ProductPageHeaderContent
+  const currentProduct = useCurrentProduct()
+  const LeftSideHeaderContent = !currentProduct
+    ? HomePageHeaderContent
+    : ProductPageHeaderContent
```

Comment:
🚧 Could this be flipped around to not have the `!`? I missed it immediately and it doesn't feel necessary with a ternary since it's easy to swap things around.

```suggestion
  const LeftSideHeaderContent = currentProduct
    ? ProductPageHeaderContent
    : HomePageHeaderContent
```

Reply:
💖 Great suggestions here, and appreciate the option to defer! ✅ Updated with the above changes, markup does feel much cleaner ✨